build_tools/build_package.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Traditionally the root directory of a github repo
+# contains all python requirements to create a distribution
+# This file setups the environment and copy over the required
+# files to the build/python_package directory that will contain
+# the python distribution directory
+sudo apt-get install -y build-essential
```

Comment:
Can these commands actually be shared with the commands in `.github/workflow/pytest.yml`? They appear to be almost identical.